areaOfInterest/areas-of-interest.ts

```diff
@@ -0,0 +1,103 @@
+class AreasOfInterest {
+
+    preInit(config: any) {
+        this.config = config;
+    }
+
+    init(api: any) {
+        console.log(api);
+        //initialize attributes
+        let map = api; // map object that rv-extension loads into this file
+        let mapConfig = this.config;
+        let zones = mapConfig.map.components.areaOfInterest._source.areas;
+        let noPic = mapConfig.map.components.areaOfInterest._source.noPicture;
```

Comment:
Is it possible to avoid the use of private properties here, `_source` should be avoided in an extension.

Reply:
See above

Reply:
update: this does not apply anymore because `areaOfInterest`  configs are how defined in a script tag